tests/migration_test.go

```diff
@@ -3699,11 +3698,11 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 			vmi, err = virtClient.VirtualMachineInstance(vmi.Namespace).Get(vmi.Name, &metav1.GetOptions{})
 			Expect(err).ToNot(HaveOccurred(), "should have been able to retrive the VMI instance")
 			cpuSetTarget := getVirtLauncherCPUSet(vmi)
-			Expect(reflect.DeepEqual(cpuSetSource, cpuSetTarget)).To(BeFalse(), "source CPUSet %v should be != than target CPUSet %v", cpuSetSource, cpuSetTarget)
+			Expect(cpuSetSource).NotTo(Equal(cpuSetTarget), "source CPUSet %v should be != than target CPUSet %v", cpuSetSource, cpuSetTarget)
```

Comment:
Do we require the print now? `"source CPUSet %v should be != than target CPUSet %v", cpuSetSource, cpuSetTarget)` ?

Reply:
I'm not sure how the `gomega` will print this by default, but you are right, this seems unnecessary.

I'll remove that.

Reply:
This is what you get 
```
  Expected
      <[]int | len:2, cap:2>: [1, 2]
  not to equal
      <[]int | len:2, cap:2>: [1, 2]
```

So I would just leave the comment(`CPUSet of source launcher should not match targets one`).

Reply:
Done :)